test/org/sagebionetworks/bridge/stormpath/StormpathAccountDaoTest.java

```diff
@@ -135,13 +136,6 @@ public void getStudyPagedAccounts() {
             accounts = accountDao.getPagedAccountSummaries(study, 0, 20, "bridgeit@");
             assertEquals(1, accounts.getItems().size());
             assertEquals("bridgeit@sagebase.org", accounts.getItems().get(0).getEmail());
-            
-            accounts = accountDao.getPagedAccountSummaries(study, 0, 20, "bridge-testing+SADT");
-            assertTrue(accounts.getItems().size() > 0);
-            for (AccountSummary summary : accounts.getItems()) {
-                assertNull(summary.getFirstName());
-                assertNull(summary.getLastName());
-            }
```

Comment:
Yes. This almost seems like an error I made when merging, this shouldn't be removed. Will restore.